x-pack/plugins/cases/common/api/cases/index.ts

```diff
@@ -5,10 +5,4 @@
  * 2.0.
  */
 
-export * from './case';
-export * from './comment';
-export * from './status';
 export * from './constants';
```

Comment:
Feels kind of a waste when all that is left is a file with

`export const OWNER_FIELD = 'owner';`

Reply:
Maybe we can move that to a new file `x-pack/plugins/cases/common/constants/cases.ts`?

Reply:
Ok!